README.md

```diff
@@ -10,59 +10,28 @@
 Welcome to the landing page of Opaque SQL! Opaque SQL is a package for Apache Spark SQL that enables processing over encrypted DataFrames using the OpenEnclave framework. 
 
 ### Quick start
-To quickly get started with Opaque SQL, you can download our Docker image (also includes other open source projects in the MC<sup>2</sup> project).
+To quickly get started running Opaque SQL, please refer to this [usage section](https://mc2-project.github.io/client-docs/opaquesql_usage.html). Note that Opaque SQL requires the [MC<sup>2</sup> client](https://github.com/mc2-project/mc2) in order to securely run an encrypted query.
```

Comment:
Does it make sense to link to the MC2 quickstart here? We're already linking to the usage section below.